docs/targets/solana.rst

```diff
@@ -1,23 +1,52 @@
 Solana
 ======
 
-The Solana target requires `Solana <https://www.solana.com/>`_ v1.8.1.
-
-Solana has the following differences to Ethereum Solidity:
-
-- The address type is 32 bytes, not 20 bytes. This is what Solana calls an "account"
-- An address literal has to be specified using the ``address"36VtvSbE6jVGGQytYWSaDPG7uZphaxEjpJHUUpuUbq4D"`` syntax
-- There is no ``ecrecover()`` builtin function, but there is a ``signatureVerify()`` function which can check ed25519
-  signatures.
-- There is no concept of gas, so there is no gas functions
-- Solana balance is stored in a ``uint64``, so *address* ``.balance``, ``.transfer()`` and ``.send()``
-  all use ``uint64`` rather than ``uint256``.
-- Solana uses different accounts for the program code, and the contract data.
-- Programs are upgradable. There is no need to implement upgrades in Solidity.
+Solana Overview
+_______________
+
+The Solana blockchain has a unique account model that resembles a key-value database.
```

Comment:
What does this mean or what are you trying to say? I don't know how Solana resembles a key-value database.

Reply:
Well that's one way of describing the database. How is that distinct from any other blockchain? Why does it matter?

Reply:
I have removed the first sentence.